bridges/FacebookBridge.php

```diff
@@ -186,19 +192,19 @@ private function collectGroupData() {
 
 		$this->groupName = $this->extractGroupName($html);
 
-		$posts = $html->find('div.userContentWrapper')
+		$posts = $html->find('div.story_body_container')
 			or returnServerError('Failed finding posts!');
 
 		foreach($posts as $post) {
 
 			$item = array();
 
-			$item['uri'] = $this->extractGroupURI($post);
-			$item['title'] = $this->extractGroupTitle($post);
-			$item['author'] = $this->extractGroupAuthor($post);
-			$item['content'] = $this->extractGroupContent($post);
-			$item['timestamp'] = $this->extractGroupTimestamp($post);
-			$item['enclosures'] = $this->extractGroupEnclosures($post);
+			$item['uri'] = $this->extractGroupPostURI($post);
+			$item['title'] = $this->extractGroupPostTitle($post);
+			$item['author'] = $this->extractGroupPostAuthor($post);
+			$item['content'] = $this->extractGroupPostContent($post);
+			$item['timestamp'] = $this->extractGroupPostTimestamp($post);
+			$item['enclosures'] = $this->extractGroupPostEnclosures($post);
```

Comment:
Why do we need to rename methods?

Reply:
I felt it made things more clear, and given they're internal methods, it shouldn't make any difference elsewhere. 
`extractGroupName()` and `extractGroupTitle()` have very similar and confusing method names otherwise. It took me a moment to realise the former is the group title, and the latter is the group-post title

Reply:
Fair enough